### ISSUE TITLE
Issue Template Updates for Figma link 

### ISSUE THREAD

**Issue by thisiskaransgit** [layer5io/meshery-service-mesh-patterns-action#6]
### Current Behavior
The project's current issue templates are missing an open invitation link where new contributors can join the community's Figma team and view user interface designs and other UX projects.

### Desired Situation
Each template that has a reference to Figma in its resources section should an invite link added.

### Implementation 
```
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
```

**Acceptance Tests**
All references to Figma include the "open invite" link.

---
**[Contributor Guide](https://docs.meshery.io/project/contributing)**

**Comment by csengineer1990** [comment]
Hi, I am a newcomer, I want to learn and grow.  can I join to solve the issue with guidance? 
but I am new to Figma with guidance definitely i will complete the task.

**Comment by Ghat0tkach** [comment]
There you go @csengineer1990  , all you need to do is add 
this 
`` 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)``
in existing issue template under .github folder

**Comment by csengineer1990** [comment]
@Ghat0tkach sir, I am new to GitHub and I tried many ways to solve this in the middle I thought I lost the commit by mistake I ran the reset command and then somehow managed to get back to revert the original commit. Now I again raise PR still facing the same problem. Please help me to complete this request.

**Comment by Ghat0tkach** [comment]
> @Ghat0tkach sir, I am new to GitHub and I tried many ways to solve this in the middle I thought I lost the commit by mistake I ran the reset command and then somehow managed to get back to revert the original commit. Now I again raise PR still facing the same problem. Please help me to complete this request.

No worries @csengineer1990 sir , do the changes again and raise a PR.
Please be sure to sign off commit this time.
Signing of commits could be definately challenge for first time. 
If you ran into any problem again , please text on slack 🙌

**Comment by csengineer1990** [comment]
yes, sir A little challenging. I am trying every possible solution which I am finding related to signing off with GPG.

**Comment by Ghat0tkach** [comment]
> yes, sir A little challenging. I am trying every possible solution which I am finding related to signing off with GPG. ok, sir.

Please don't be worry , All you have to is  after commiting your changes , the commit messaage which you use to write  using 
  ``git commit -m "msg"``
just replace it with
``git commit -s -m "msg"``

**Comment by abhijeetgauravm** [comment]
I want to work on it. Can u assign me?
